eng/tools/specs-model/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@azure-tools/specs-model",
+  "private": true,
+  "type": "module",
+  "main": "dist/src/index.js",
+  "bin": {
+    "get-specs-model": "cmd/get-specs-model.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "postinstall": "npm run build",
+    "test": "vitest",
+    "test:ci": "vitest run --coverage --reporter=verbose",
+    "lint": "eslint . -c eslint.config.js --report-unused-disable-directives --max-warnings 0",
+    "lint:fix": "eslint . -c eslint.config.js --fix",
+    "prettier": "prettier . --check",
+    "prettier:debug": "prettier . --check ---log-level debug",
+    "prettier:write": "prettier . --write"
+  },
+  "engines": {
+    "node": ">= 18.0.0"
+  },
+  "dependencies": {
+    "lodash": "^4.17.21"
```

Comment:
Where is lodash used in your code?  I didn't see it.

Reply:
Not yet used, but I am quite sure I will use it. But I can remove it for now.

Reply:
If eslint (or any similar tools) could enforce "every dependency must actually be used", I'd support enabling the check.